crypto/ec/ecp_nistz.c

```diff
@@ -40,14 +40,14 @@ void gfp_little_endian_bytes_from_scalar(uint8_t str[], size_t str_len,
     str[i + 0] = d & 0xff;
     str[i + 1] = (d >> 8) & 0xff;
     str[i + 2] = (d >> 16) & 0xff;
-    str[i + 3] = (d >>= 24) & 0xff;
-    if (BN_BYTES == 8) {
-      d >>= 8;
-      str[i + 4] = d & 0xff;
-      str[i + 5] = (d >> 8) & 0xff;
-      str[i + 6] = (d >> 16) & 0xff;
-      str[i + 7] = (d >> 24) & 0xff;
-    }
+    str[i + 3] = (d >> 24) & 0xff;
+#if BN_BYTES == 8
```

Comment:
That was what I did first, but then you get a warning-as-error about shifting by the width of the integer. I could try to trick that warning if you prefer.